notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -3186,6 +3186,63 @@ the scientist profession [Dr. Herbert West and Dr. Howard Phillips were experime
+extraterrestrial being [The film's main character Byeong-gu ultimately failed in his effort to stop a race of humanoid aliens from the Andromeda galaxy from attacking Earth.]
+the desire for vengeance [The viewer is led to believe that a vengeance fueled psychosis caused Byeong-gu to kidnap a corporate executive and torture him to the brink of death.]
+human vs. captivity [Kang was abducted, imprisoned in a basement workshop and tortured to the brink of death.]
+boyfriend and girlfriend [Byeong-gu and his unwaveringly loyal, circus-performer girlfriend Su-ni.]
+romantic love [Su-ni was in love with and unwaveringly loyal to her deranged boyfriend Byeong-gu.]
+alien living secretly among us [A revelation of the film is that the corporate executive Kang was actually the leader of an alien race from the Andromeda galaxy, and himself one of their kind.]
+overactive imagination [The viewer is led to believe that Byeong-gu was committed to the fantastical conviction that his captive Kang was a hostile space alien, although in the end his zany belief was proved correct.]
+law enforcement [Detective Choo and Detective Kim were investigating the abduction of the corporate executive Kang.]
+torture [Kang was imprisoned in a workshop basement and subjected to numerous gruesome tortures.]
+coping with being tortured [Kang was imprisoned in a workshop basement and subjected to numerous gruesome tortures.]
+delusional point of view [The film is shown from the point of view of a deranged man, named Byeong-gu, who has kidnapped another man whom he is convinced is a space alien.]
+the making of a monster to society [The story explores how Byeong-gu's traumatic past led him to become a deranged and sadistic killer.]descent into madness [The story explores how Byeong-gu slowly went mad from all the violence that surrounded him.]
+aliens meddling in history [A revelation of the story is that aliens from Andromeda had created humans in their own image and had been guiding the development of humanity since prehistoric times.]
+sadism [Byeong-gu was exceeding sadistic in his treatment of his captive, Kang. Among other things, Byeong-gu delighted in crucifying Kang and crushing his shin with the back of an axe.]
+
+:: Minor Themes
+telepathic ability [Byeong-gu suppressed Kang's ability to communicate telepathically by shaving Kang's head.]
+master and pet [Byeong-gu fed the remains of Detective Choo to Byeong-gu' own dog, named Earth.]
+mother and son [Byeong-gu didn't want to let go of his comatose mother, and refused to pull the plug on her. Byeong-gu conversed with his ill mother in a flashback scene.]
+coping with the death of a parent [Byeong-gu was devastated when Kang tricked him into giving a lethal dose of benzene to Byeong-gu's own comatose mother.]
+human interpersonal relationship [Detective Kim was assigned to work on the Kang abduction case with together with Detective Choo, although Choo died before they had much of a chance to bond.]
```

Comment:
We've discussed it before but perhaps introducing a "close partners at work" kknx of theme would be in order?

Reply:
This is, by my count, the eighth instance of using **human interpersonal relationship** for what amounts to a pair of "buddy cops". I'll leave this theme as is for now, but I concur with introducing a "close parters at work" (or perhaps the more specific "partners on the force") theme at some point.